app/controllers/animals_controller.rb

```diff
@@ -0,0 +1,45 @@
+class AnimalsController < ApplicationController
+
+    def index
+        animals = Animal.all
+        render json: animals
+    end
+
+    def show
+        animal = Animal.find(params[:id])
+        render :json => animal.to_json(:include => :sightings)
+    end
+
+    def create
+        animal = Animal.create (animal_params)
+        if animal.valid?
+            render json: animal
+        else 
+            render json: animal.errors
+        end
+    end
+
+    def destroy
+        animal = Animal.find(params[:id])
+        if animal.destroy
+            render json:animal
+        else 
+            render json:errors
+        end
+    end
+
+    def update
+        animal = Animal.find (params[:id])
+        animal.update(animal_params)
+        if animal.valid?
+            render json:animal
+        else
+            render json:errors
+        end
+    end
+    
```

Comment:
Strong params method is correct. Permits the correct parameters and is all encapsulated in a private method. Nice job!